app/models/state_file_w2.rb

```diff
@@ -3,6 +3,9 @@
 # Table name: state_file_w2s
 #
 #  id                          :bigint           not null, primary key
+#  employee_name               :string           not null
+#  employee_ssn                :string
+#  employer_name               :string           not null
```

Comment:
[Sand] Noting that we discussed adding `WagesAmt` and `WithholdingAmt` here as well, but fine to leave for now and follow up later.